frontends/mit-open/src/common/routes.tsx

```diff
@@ -1,22 +1,18 @@
 import React from "react"
 import { RouteObject, Outlet } from "react-router"
-import HomePage from "./pages/Home/Home"
-import RestrictedRoute from "./components/RestrictedRoute"
-import LearningPathListingPage from "./pages/learningpaths/LearningPathListingPage"
-import LearningPathDetailsPage from "./pages/learningpaths/LearningPathDetails"
-import ArticleDetailsPage from "./pages/articles/ArticleDetailsPage"
-import {
-  ArticlesCreatePage,
-  ArticleEditingPage,
-} from "./pages/articles/ArticleUpsertPages"
-import ErrorPage from "./pages/errors/ErrorPage"
-import * as urls from "./pages/urls"
-import * as deprecatedUrls from "./infinite-pages/urls"
-import EditFieldPage from "./infinite-pages/field-details/EditFieldPage"
-import FieldPage from "./infinite-pages/field-details/FieldPage"
-
-import Header from "./components/Header"
-import { Permissions } from "./util/permissions"
+import HomePage from "pages/HomePage/HomePage"
+import RestrictedRoute from "components/RestrictedRoute/RestrictedRoute"
+import LearningPathListingPage from "pages/LearningPathListingPage/LearningPathListingPage"
+import LearningPathDetailsPage from "pages/LearningPathDetailsPage/LearningPathDetailsPage"
+import ArticleDetailsPage from "pages/ArticleDetailsPage/ArticleDetailsPage"
+import { ArticleCreatePage, ArticleEditPage } from "pages/ArticleUpsertPages"
```

Comment:
We have the option of grouping pages that are similar and providing an index.ts facade for named import/exports. In this case both `ArticleCreatePage` and `ArticleEditPage` import and wrap inside `ArticleUpsertPage` so made sense to self contain.  Will check whether they are imported anywhere and otherwise delete.

Reply:
I think I just misread the imports + missed the index file.